src/strategies/dotnet-yoshi.ts

```diff
@@ -0,0 +1,142 @@
+  {type: 'feat', section: 'New features'},
+  {type: 'fix', section: 'Bug fixes'},
+  {type: 'perf', section: 'Performance improvements'},
+  {type: 'revert', section: 'Reverts'},
+  {type: 'chore', section: 'Miscellaneous chores', hidden: true},
+  {type: 'docs', section: 'Documentation improvements'},
+  {type: 'style', section: 'Styles', hidden: true},
+  {type: 'refactor', section: 'Code Refactoring', hidden: true},
+  {type: 'test', section: 'Tests', hidden: true},
+  {type: 'build', section: 'Build System', hidden: true},
+  {type: 'ci', section: 'Continuous Integration', hidden: true},
+];
+const DEFAULT_CHANGELOG_PATH = 'docs/history.md';
+const DEFAULT_PULL_REQUEST_TITLE_PATTERN =
+  'Release${component} version ${version}';
+const RELEASE_NOTES_HEADER_PATTERN =
+  /#{2,3} \[?(\d+\.\d+\.\d+-?[^\]]*)\]?.* \((\d{4}-\d{2}-\d{2})\)/;
+
+interface Api {
+  id: string;
+  version: string;
+  noVersionHistory?: boolean;
+  shortName: string;
+}
+interface ApisJsonConfiguration {
+  apis: Api[];
+}
+
+export class DotnetYoshi extends BaseStrategy {
+  constructor(options: BaseStrategyOptions) {
+    options.changelogSections = options.changelogSections ?? CHANGELOG_SECTIONS;
+    options.changelogPath = options.changelogPath ?? DEFAULT_CHANGELOG_PATH;
+    options.pullRequestTitlePattern =
+      options.pullRequestTitlePattern ?? DEFAULT_PULL_REQUEST_TITLE_PATTERN;
+    options.includeVInTag = options.includeVInTag ?? false;
+    super(options);
+  }
+
+  protected async buildReleaseNotes(
+    conventionalCommits: ConventionalCommit[],
+    newVersion: Version,
+    newVersionTag: TagName,
+    latestRelease?: Release
+  ): Promise<string> {
+    const notes = await super.buildReleaseNotes(
+      conventionalCommits,
+      newVersion,
+      newVersionTag,
+      latestRelease
+    );
+    return notes.replace(
+      RELEASE_NOTES_HEADER_PATTERN,
+      '## Version $1, released $2'
+    );
+  }
+
+  private async getApi(): Promise<Api | undefined> {
+    try {
+      const contents = await this.github.getFileContentsOnBranch(
+        'apis/apis.json',
+        this.targetBranch
+      );
+      const apis = JSON.parse(contents.parsedContent) as ApisJsonConfiguration;
+      const component = await this.getComponent();
+      return apis.apis.find(api => api.id === component);
+    } catch (e) {
+      if (e instanceof FileNotFoundError) {
+        throw new MissingRequiredFileError(
+          'apis/apis.json',
+          DotnetYoshi.name,
+          `${this.repository.owner}/${this.repository.repo}`
+        );
+      }
+      throw e;
+    }
+  }
+
+  protected async buildUpdates(
+    options: BuildUpdatesOptions
+  ): Promise<Update[]> {
+    const updates: Update[] = [];
+    const version = options.newVersion;
+    const component = await this.getComponent();
+
+    const api = await this.getApi();
+    if (api?.noVersionHistory) {
+      logger.info(
+        `Skipping changelog for ${component} via noVersionHistory configuration`
```

Comment:
Is `apis.json` similar to `release-please-manifest.json`?

Wondering out loud if an alternate implementation would have been treating .NET as being configured as a manifest based release, and supporting this alternate metadata file rather than `releasae-please-manifest.json`.

Reply:
It will still be a manifest based release, but we don't need to have extra configs for this flag, unless it's a feature (skipping the changelog) we want to support for more strategies.